llvm/test/CodeGen/NVPTX/i128-array.ll

```diff
@@ -0,0 +1,25 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -O0 -march=nvptx64 -mcpu=sm_20 | FileCheck %s
+
+define byval [2 x i128] @foo(i64 %a, i32 %b) {
```

Comment:
I meant that we should have a test for a `byval [2 x 128]` parameter.

Reply:
Sorry for delay, done